apps/site/pages/components/molecules/breadcrumb.mdx

```diff
@@ -104,7 +130,7 @@ import '@faststore/ui/src/components/molecules/Breadcrumb/styles.scss'
 
 ## Props
 
-<PropsSection name="Breadcrumb" />
+<BreadcrumbPropsSection component="Breadcrumb" />
```

Comment:
It didn't work because we need to change how we export this component props. 
Automatic props don’t work when declaring props with type. We have to use interface instead to it works appropriately. Lemme try to change it for u.

<img width="961" alt="image" src="https://github.com/vtex/faststore/assets/3356699/f4ebe1a2-127b-434c-b901-16056aad4a62">

Reply:
@hellofanny, not sure what am I doing wrong but I believe is something related to the const we are declaring in `Breadcrumb.tsx` and I keep getting an error when I use `interface`. 🤔

Reply:
Can u try like this pls?

`export interface BreadcrumbBaseProps extends Omit< BreadcrumbBaseProps, 'isDesktop'> { }`

Reply:
This case is a little cumbersome, because we need to change all breadcrumb related Props to export interface. I took the liberty to [change here](https://github.com/vtex/faststore/pull/1782/commits/a754236a75b6796017b716f51811a978593c4ecf) :)